cmd/internal/apidump/apidump.go

```diff
@@ -201,6 +219,28 @@ func init() {
 		"",
 		"Your Akita project. Exactly one of --out or --project must be specified.")
 
+	Cmd.Flags().StringVar(
+		&postmanCollectionID,
+		"collection",
+		"",
+		"Your Postman collectionID. Both --collection and --key must be specified.")
+
+	Cmd.MarkFlagsMutuallyExclusive("out", "project", "collection")
+
+	Cmd.Flags().StringVar(
+		&postmanAPIKey,
+		"key",
+		"",
+		"Your Postman API Key. Both --collection and --key must be specified.")
+
+	Cmd.MarkFlagsRequiredTogether("collection", "key")
```

Comment:
I think that it would be best if we did not accept sensitive material on the command line. We might have to get Paul to update the command line in the instructions from

`akita kube inject --collection foo --key <key>`

to 

`POSTMAN_API_KEY=<key> akita kube inject --collection foo`

Reply:
What advantage will we be having by accepting key as env var? In both cases user have to specify the key in command line for `kube inject`.

Why can't we ask user to run `akita login` first, just like we are doing earlier?

Reply:
The advantage here is that the actual command arguments are visible in contexts where the environment variables (even if specified on the shell) are not -- it's a security best practice.  There is less potential for exposing sensitive information.

(We could go back and rewrite the command line on initialization; some tools do that.)

Our goal here is to have a single command that they can copy from the UI and run.  Adding a second step, particularly one that's interactive like `login`, interferes with that.